packages/react-components/react-search/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@fluentui/react-search",
+  "version": "9.0.0-alpha.0",
+  "private": true,
+  "description": "React components for building web experiences",
```

Comment:
Would be good to add a semi-descriptive line here, along the lines of "Search input for Fluent UI v9"

Reply:
will do!